extensions/machine-learning-services/src/packageManagement/packageManagementService.ts

```diff
@@ -9,8 +9,9 @@ import { QueryRunner } from '../common/queryRunner';
 import * as constants from '../common/constants';
 import { ApiWrapper } from '../common/apiWrapper';
 import * as utils from '../common/utils';
+import * as nbExtensionApis from '../typings/notebookServices';
 
-export class ServerConfigManager {
+export class PackageManagementService {
```

Comment:
its not used outisde ML ext? I am worried if this namechange would impact anything else?

Reply:
no it's not. well nothing should use module from inside extension directly. it's not exposed as service also

---
In reply to: [393906056](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r393906056) [](ancestors = 393906056)